sources/packages/backend/libs/utilities/src/config/config.service.ts

```diff
@@ -154,6 +154,50 @@ export class ConfigService {
     );
   }
 
+  /**
+   * Maximum idle time for warning student Configuration.
+   * @returns maximum idle time for warning student number.
+   */
+  get maximumIdleTimeForWarningStudent(): number {
+    return this.getCachedConfig(
+      "maximumIdleTimeForWarningStudentConfig",
+      +process.env.maximumIdleTimeForWarningStudent,
+    );
+  }
+
+  /**
+   * Maximum idle time for warning supporting user Configuration.
+   * @returns maximum idle time for warning supporting user number.
+   */
+  get maximumIdleTimeForWarningSupportingUser(): number {
+    return this.getCachedConfig(
+      "maximumIdleTimeForWarningSupportingUserConfig",
+      +process.env.maximumIdleTimeForWarningSupportingUser,
+    );
+  }
+
+  /**
+   * Maximum idle time for warning institution Configuration.
+   * @returns maximum idle time for warning institution number.
+   */
+  get maximumIdleTimeForWarningInstitution(): number {
+    return this.getCachedConfig(
+      "maximumIdleTimeForWarningInstitutionConfig",
+      +process.env.maximumIdleTimeForWarningInstitution,
+    );
+  }
+
+  /**
+   * Maximum idle time for warning AEST Configuration.
+   * @returns maximum idle time for warning AEST number.
+   */
+  get maximumIdleTimeForWarningAEST(): number {
+    return this.getCachedConfig(
+      "maximumIdleTimeForWarningAestConfig",
```

Comment:
AEST must be in upper case.

Reply:
I still can't see the change. The comment is on line 196 (`maximumIdleTimeForWarningAestConfig`).